src/Configuration/src/Kubernetes/KubernetesConfigurationBuilderExtensions.cs

```diff
@@ -13,62 +14,99 @@ namespace Steeltoe.Extensions.Configuration.Kubernetes;
 public static class KubernetesConfigurationBuilderExtensions
 {
     /// <summary>
-    /// Add configuration providers for ConfigMaps and Secrets.
+    /// Adds Kubernetes configuration providers for config maps and secrets.
```

Comment:
```suggestion
    /// Adds configuration providers for Kubernetes ConfigMaps and Secrets.
```
These are Kubernetes objects, I think they are supposed to be capitalized
https://kubernetes.io/docs/concepts/configuration/configmap/
https://kubernetes.io/docs/concepts/configuration/secret/